main.js

```diff
@@ -76,21 +77,76 @@ function login() {
      * @todo 1. Write code for form validation.
      * @todo 2. Fetch the auth token from backend and login the user.
      */
-}
+    const username = document.getElementById('inputUsername').value.trim();
+    const password = document.getElementById('inputPassword').value;
+
+    if (username != '' || password != '')  {
```

Comment:
Incorrect Validation condition, it will result true `true` even if one of username and password is filled.